### DISCUSSION TITLE
Browser support?

### DISCUSSION THREAD

**Discussion by heauton** [supabase/supabase#1784] [Questions] [answered]
Does anyone know browser support for the JS library? I read https://supabase.io/blog/2020/12/13/supabase-dashboard-performance#using-native-browser-apis going against IE 11 but other than that, no versions specified.

**Comment by soedirgo** [comment, marked as answer]
Hey @heauton, the JS library is isomorphic so you can use it in the browser. The following HTML should work fine:
```html
<!doctype html>
<script type="module">
 import { createClient } from 'https://cdn.skypack.dev/@supabase/supabase-js'
 const supabase = createClient('https://yourprojectref.supabase.co', 'your.api.token')
 supabase.from('your_table').select().then(console.log)
</script>
```

Edit: I may have misinterpreted your question. We try to support the last 3 major versions of Chrome, Firefox, and Safari - let us know if the library doesn't work in these versions.
